meta-iot2000-example/scripts/bin/extract_license

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Extract and combine license file
+# Eg:
+#    extract_license build/tmp/deploy/licenses license.dat
+#
```

Comment:
Copyright and license header is missing. We didn't use that consistently so far, specifically leaving trivial one or two-line files untagged, but we should do better. Grab the pattern from meta-iot2000-bsp/recipes-tools/switchmode/files/switchmode.c e.g.